.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: Default CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Setup Nodejs
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: Install dependencies
+      run: npm ci
+    - name: Lint
+      run: npm run lint
+    - name: Test
```

Comment:
It's too bad we can't split these out into separate "jobs" without copying all that boilerplate...

Reply:
I just copied these files from frontend-build, to be clear.  Maybe I'm not following the right process?

Reply:
Nah, sorry, you're good.

This works; it runs all the tests in serial and reports status as a single job.
To have multiple job "checks" (eslint, tests, commitlint) each with their own status and run in parallel, one could write this workflow to contain multiple entries in the `jobs` dict (there's currently just one, `build`), but it requires copy/pasting all of that setup boilerplate.. or writing a common, parameterized build script (Makefile, bash, etc.) that handles it.

Again, this is good as written, just lamenting that Github Actions don't appear to have a "common_setup" stage/step.